lib/rules/lint-no-childless-elements.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/*
+ Disallows self-closing void elements
+
+ ```
+ {{! good }}
+ <hr>
+
+ {{! bad}}
+ <hr />
+ ```
+
+ The following values are valid configuration:
+
+   * boolean -- `true` for enabled / `false` for disabled
+ */
+
+const Rule = require('./base');
+const createErrorMessage = require('../helpers/create-error-message');
+
+/**
+ * [Specs of Void Elements]{@link https://www.w3.org/TR/html-markup/syntax.html#void-element}
+ * @type {Object}
+ */
+const BLOCK_TAGS = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p', 'span', 'a', 'ul', 'ol', 'li', 'div'];
+
+module.exports = class NoChildlessElements extends Rule {
+  parseConfig(config) {
```

Comment:
This rule has no extra configuration so this entire `parseConfig` function should be omitted.